e2e/playwright/point-click.spec.ts

```diff
@@ -1,33 +1,59 @@
-import { test, expect, AuthenticatedApp } from './fixtures/fixtureSetup'
+import { test, expect, Page } from './zoo-test'
 import { EditorFixture } from './fixtures/editorFixture'
 import { SceneFixture } from './fixtures/sceneFixture'
 import { ToolbarFixture } from './fixtures/toolbarFixture'
+import fs from 'node:fs/promises'
+import path from 'node:path'
+import { getUtils } from './test-utils'
 
 // test file is for testing point an click code gen functionality that's not sketch mode related
 
-test(
-  'verify extruding circle works',
-  { tag: ['@skipWin'] },
-  async ({ app, cmdBar, editor, toolbar, scene }) => {
-    test.skip(
-      process.platform === 'win32',
-      'Fails on windows in CI, can not be replicated locally on windows.'
-    )
-    const file = await app.getInputFile('test-circle-extrude.kcl')
-    await app.initialise(file)
-    const [clickCircle, moveToCircle] = scene.makeMouseHelpers(582, 217)
+test('verify extruding circle works', async ({
+  context,
+  homePage,
+  cmdBar,
+  editor,
+  toolbar,
+  scene,
+}) => {
+  // TODO: fix this test on windows after the electron migration
+  test.skip(process.platform === 'win32', 'Skip on windows')
+  const file = await fs.readFile(
+    path.resolve(
+      __dirname,
+      '../../',
+      './src/wasm-lib/tests/executor/inputs/test-circle-extrude.kcl'
+    ),
+    'utf-8'
+  )
+  await context.addInitScript((file) => {
+    localStorage.setItem('persistCode', file)
+  }, file)
```

Comment:
I just ported what each test does as I went along; sure, it can be done either way. Most though preferred to persistCode via localStorage because they were web tests